src/shells/webextension/injectGlobalHook.js

```diff
@@ -1,10 +1,6 @@
+// Simply importing this file will install the global hook here
 import installGlobalHook from '../../backend/utils/installGlobalHook';
 
-const script = document.createElement('script');
-script.textContent = `;(${installGlobalHook.toString()}(window))`;
-document.documentElement.appendChild(script);
-script.parentNode.removeChild(script);
-
```

Comment:
It looks like this change causes webpack to tree shake the hook from the build.

Reply:
Are you seeing it fail to load? I tested this a bunch and things were working fine without it.

Reply:
Yea I'm not seeing the mobx tab in dev tools using the build on chrome web store.

Reply:
Alrighty, something weird must have happened where the old hook injection got cached in my browser or something. I was able to reproduce the bug, and see it fixed in https://github.com/mobxjs/mobx-devtools/pull/133